docs/reference-guide/query-modules/implement-custom-query-modules/api/python-api.md

```diff
@@ -129,6 +129,55 @@ CALL example.procedure(“single argument”) YIELD result;
 Naturally, you may pass in different arguments.
 
 
+## class mgp.function(func: Callable[[…]])
```

Comment:
We left out the docs of the python API in Memgraph, but @jmatak wrote this by hand, thus we can include this. I will open a PR in Memgraph repo to have the same docs in `mgp.py`.

Reply:
[PR](https://github.com/memgraph/memgraph/pull/391/files) for Memgraph.